packagedb/tests/test_api.py

```diff
@@ -265,14 +265,31 @@ def setUp(self):
         self.package3 = Package.objects.create(**self.package_data3)
         self.package3.refresh_from_db()
 
+        self.package_data4= {
```

Comment:
I think you should add two new maven packages, one of them a binary package, the other a source_archive package. These packages should have the same purl fields (type, namespace, name, version) with the qualifiers being different. On the binary package, leave out license information, and on the source archive package, have license information. We want to see the enhanced package data returned for the binary package in the test results when we enable the option.